pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
Hmm, ok.  So how about the annotation is rebranded as supplemental groups and takes a string of comma delimited groups.  Groups in the format of `n-n` can be parsed into an allowable range by policy strategies.  When assigning an fs group we take the first gid or the first gid of the range.  The allocator will still default this annotation to the first uid of the uid range but an admin can customize it.

openshift.io/sa.scc.supplemental-groups = "1,2,3-10" - strategy effectively allows supplemental groups of 1-10 and uses 1 as fsgroup

openshift.io/sa.scc.supplemental-groups = "3-10" - strategy allows supplemental groups of 3-10 and uses 3 as fsgroup

openshift.io/sa.scc.supplemental-groups = "3-10,12-15" - strategy allows supplemental groups of 3-10 and 12-15 and uses 3 as fsgroup